package ru.cft.shiftlab.gorin.testtask.market.exceptions;

/**
 * Исключение, вызванное ошибкой обновления сущности в Базе Данных
 */
public class UpdatingRecordException extends Exception {
    public UpdatingRecordException() {
        super();
    }
}
